stable/luigi/templates/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "luigi.fullname" . }}
+  labels:
+    app: {{ template "luigi.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
```

Comment:
`spec.selector` is missing.

Reply:
I think your talking about `service.yaml`. if so, it does exist in that file. I'm not familiar with deployment `spec.selector`